project/iterative_sorting.py

```diff
@@ -20,8 +23,13 @@ def selection_sort( arr ):
 
 # TO-DO: implement the Insertion Sort function below
 def insertion_sort( arr ):
-
-    return arr
+  for n in range(1,len(arr)):
+    element = arr[n]
+    while n > 0 and n[-1] < element:
+      arr[n] = arr[n-1]
+        n = n-1
+        arr[n] = element
+  return arr
```

Comment:
Once you fix the other thing I commented on run a simple test in your iterative_sorting.py like so 
```
arr = [2, 5, 9, 7, 4, 1, 3, 8, 6]
print(arr)
arr = insertion_sort(arr)
print(arr)
```

You will notice that you are sorting the elements but what is wrong with the way they are sorted?